netdata.spec.in

```diff
@@ -76,20 +85,18 @@ happened, on your systems and applications.
 rm -rf $RPM_BUILD_ROOT
 %{__make} %{?_smp_mflags} DESTDIR=$RPM_BUILD_ROOT install
 
-find $RPM_BUILD_ROOT -name .keep -print0 | xargs --null --no-run-if-empty rm
+find $RPM_BUILD_ROOT -name .keep -delete
 
 install -m 644 -p system/netdata.conf $RPM_BUILD_ROOT%{_sysconfdir}/%{name}
 
 mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/logrotate.d
 install -m 644 -p system/netdata.logrotate $RPM_BUILD_ROOT%{_sysconfdir}/logrotate.d/%{name}
 
 %if %{with systemd}
-install -d $RPM_BUILD_ROOT%{_unitdir}
-install -m 644 -p system/netdata.service $RPM_BUILD_ROOT%{_unitdir}/netdata.service
+install -Dm 644 -p system/netdata.service $RPM_BUILD_ROOT%{_unitdir}/netdata.service
```

Comment:
I usually prefer the install -d -m 0755 xxx as explicit, but it is up to you guys... anyway, mkdir should go away.